src/lib/routes/admin-api/metrics.ts

```diff
@@ -90,8 +104,12 @@ class MetricsController extends Controller {
                 openApiService.validPath({
                     tags: ['Metrics'],
                     operationId: 'getApplication',
+                    summary: 'Get specific application',
+                    description:
+                        'Gets specific application with `appName`. An application contains information on name, sdkVersion and how to display it in a list.',
```

Comment:
What is `sdkVersion` here?

```suggestion
                        'Returns data about the specified application (`appName`). The data contains information on the name of the application, sdkVersion, as well as data about how to display this application in a list.',
```